content/recover.md

```diff
@@ -0,0 +1,252 @@
+
+#    Description
+237  VM for 'cloud_controller_ng/6beb143e-055b-48ce-9eff-8236866b0dc7 (1)' with cloud ID 'vm-b457a975-73e0-4330-476c-86175eda1438' is not responding.
+
+1 unresponsive_agent problems
+
+1: Skip for now
+2: Reboot VM
+3: Recreate VM without waiting for processes to start
+4: Recreate VM and wait for processes to start
+5: Delete VM
+6: Delete VM reference (forceful; may need to manually delete VM from the Cloud to avoid IP conflicts)
+unresponsive_agent (1): 2
+
+Override current max_in_flight value of '50%'? [yN]:
+
+Instance Group 'diego_cell'
+
+
+Problem type: missing_vm
+
+#    Description
+239  VM for 'diego_cell/837faa6d-735b-4068-a201-944916c3c051 (1)' with cloud ID 'vm-4a68e03f-5eb5-41b8-5ce1-2eba1f62c873' missing.
+242  VM for 'diego_cell/8a6186b1-1465-453b-b9dd-35de5dedfed9 (0)' with cloud ID 'vm-a80bf596-bf23-4e13-67c7-9219e75ca3da' missing.
+
+2 missing_vm problems
+
+1: Skip for now
+2: Recreate VM without waiting for processes to start
+3: Recreate VM and wait for processes to start
+4: Delete VM reference
+missing_vm (1): 2
+
+Problem type: mount_info_mismatch
+
+#    Description
+243  Inconsistent mount information:
+     Record shows that disk 'disk-cf4a1687-3399-4d61-4661-2974255e19c3' should be mounted on vm-bb29ea10-fe59-4feb-69dc-8917694f5ab3.
+     However it is currently :
+     	Not mounted in any VM
+
+1 mount_info_mismatch problems
+
+1: Ignore
+2: Reattach disk to instance
+3: Reattach disk and reboot instance
+mount_info_mismatch (1): 3
+
+Override current max_in_flight value of '2'? [yN]: y
+
+max_in_flight override for 'diego_cell' (2): 3
+
+Instance Group 'router'
+
+
+Problem type: missing_vm
+
+#    Description
+240  VM for 'router/408d72b9-d5d9-455c-9e6e-75997d4e47e4 (1)' with cloud ID 'vm-3b6643c4-9ffe-4e6e-4759-d796db979b20' missing.
+
+1 missing_vm problems
+
+1: Skip for now
+2: Recreate VM without waiting for processes to start
+3: Recreate VM and wait for processes to start
+4: Delete VM reference
+missing_vm (1): 2
+
+Problem type: unresponsive_agent
+
+#    Description
+238  VM for 'router/a84205fb-c6be-4c26-89f2-93f6019acaf1 (0)' with cloud ID 'vm-77fa8ee8-30cb-40cf-74f2-fa0716005322' is not responding.
+
+1 unresponsive_agent problems
+
+1: Skip for now
+2: Reboot VM
+3: Recreate VM without waiting for processes to start
+4: Recreate VM and wait for processes to start
+5: Delete VM
+6: Delete VM reference (forceful; may need to manually delete VM from the Cloud to avoid IP conflicts)
+unresponsive_agent (1): 2
+
+Override current max_in_flight value of '1'? [yN]: y
+
+max_in_flight override for 'router' (1): 100%
+
+Succeeded
+```
+
+### Recovery plan {: #recovery-plan }
+
+Each recovery plan has the following schema:
+
+**instance_groups_plan** [Array, required]: The name of instance groups in the deployment to recover.
+
+* **max_in_flight_override** [String, optional]: The `max_in_flight` value to use for problem resolution in the given instance group
```

Comment:
Would be good to specify here what valid values for this string are.

Can I put "all"? "Two"? etc, etc, etc. (As a user of this feature, if I didn't look at the example, I wouldn't know that "100%" is a valid value. What else is valid? Fractional percents? Fractions? (so on and so forth))